Project.toml

```diff
@@ -10,16 +10,17 @@ Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
 SHA = "ea8e919c-243c-51af-8825-aaa63cd721ce"
 SparseArrayKit = "a9a3c162-d163-4c15-8926-b8794fbefed2"
 Tar = "a4e569a6-e804-4fa4-b0f3-eef7a1d5b13e"
-TensorKit = "07d1fe3e-3e46-537d-9eac-e9e13d0d4cec"
+TensorKitSectors = "13a9c161-d5da-41f0-bcbd-e1a08ae0647f"
+TensorOperations = "6aa20fa7-93e2-5fca-9bc0-fbd0db3c71a2"
```

Comment:
This should only be a test dependency: It should have a [compat], [extras] and [targets] entry.